rom/laghos.cpp

```diff
@@ -1581,8 +1581,7 @@ int main(int argc, char *argv[])
             romOper[0]->ApplyHyperreduction(romS);
         }
 
-		// To perform the induced Gram-Schmidt during the online stage, so that
-		// the reduced mass matrices are identity.
+		// TODO: do we want that for the energy-conserving EQP?
```

Comment:
Note my question here: do we want to call `ApplyHyperreduction` for the energy conserving EQP case? 

This function calls the induced Gram-Schmidt method, which we're not using anymore (since the Gram-Schmidt flag is now false). Then it calls a function `ComputeReducedMatrices` which I'm not sure about what it computes exactly. Among other things, I think it computes the matrix needed to get the RHS of the position equation (the Lagrangian part of the problem).